modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/jaxrs/container/request/filter/ObjectDefinitionIdContainerRequestFilter.java

```diff
@@ -50,20 +46,24 @@ public void filter(ContainerRequestContext containerRequestContext) {
 			uriInfo.getQueryParameters();
 
 		queryParameters.add(
-			"objectDefinitionId", String.valueOf(_objectDefinitionId));
+			"objectDefinitionId",
+			String.valueOf(_objectDefinition.getObjectDefinitionId()));
 
 		for (Map.Entry<String, List<String>> entry :
 				queryParameters.entrySet()) {
 
 			uriBuilder.queryParam(entry.getKey(), entry.getValue());
 		}
 
-		uriBuilder.queryParam("taskItemDelegateName", _objectDefinitionName);
+		uriBuilder.queryParam(
+			"taskItemDelegateName",
+			_objectDefinition.getShortName() +
```

Comment:
Does this has to be the same key used in the deployer? if so can we compute it in a util class once and reuse in both places?

Reply:
🤔 🤔 🤔 🤔 
Yeah! Good idea